makeservices/makemysql

```diff
@@ -4,20 +4,54 @@ if [[ "$(hostname)" != "tsunami" && "$(hostname)" != "dev-tsunami" ]]; then
     exit 1
 fi
 
-PASS=$(sudo -u mysql /opt/share/utils/makeservices/makemysql-real | tee /dev/tty | tail -n 1 | grep -Po '(?<=: )([0-9a-zA-Z]){24,}$')
+IGNORE_WP=false
+UNREADABLE_OUTPUT=false
+SILENT=""
+
+# Parse arguments down below
+while (( "$#" )); do
+    case "$1" in
+        -i|--ignore-wp)
+            IGNORE_WP=true
+            shift
+            ;;
+        -u|--unreadable-output)
+            UNREADABLE_OUTPUT=true
+            shift
+            ;;
+        -s|--silent)
+            YES="--silent"
+            shift
+            ;;
+        # I've seen -*|--*=), but idk what does the = serve
+        -*|--*)
+            # Output to stderr
+            echo "Error: Unsupported flag $1" >&2
+            exit 1
+            ;;
+    esac
+done
+
+PASS=$(sudo -u mysql /opt/share/utils/makeservices/makemysql-real $SILENT | tail -n 1)
 
 # Check if wp is installed with wp-cli
 # And change password if installed with easywp
 # But change password only if installed with easywp is a wrong statement
 # Use --ignore-wp flag to skip this process
-if (wp core is-installed --quiet --path="$HOME/public_html" > /dev/null 2>&1) &&
-        ([ $# -ne 1 ] || ( [ $# -eq 1 ] && [ ! "$1" = '--ignore-wp' ])); then
-    echo "WordPress installation detected, changing WordPress mysql password for you."
+if (wp core is-installed --quiet --path="$HOME/public_html" > /dev/null 2>&1) && ! $IGNORE_WP ; then
```

Comment:
this is kind of weird, it is literally running the commands `true` or `false`

Reply:
lemme do some research, my bash sucks

Reply:
- I'd check `IGNORE_WP` first before running a command.
 - Nit: beware that putting parentheses around a command runs it in a subshell (which is not necessary in this case). Up to you if you want to keep them around or not, though; I don't think it makes too much of a practical difference in this case either.
 - https://stackoverflow.com/questions/2953646/how-can-i-declare-and-use-boolean-variables-in-a-shell-script